src/scss/root/_index.scss

```diff
@@ -164,6 +164,15 @@ $component-map: get-all-component-definitions($tokens, $breakpoints);
 					@if type-of($cValue) != map {
 						$new-value: replace-token-reference($cValue);
 						--#{$prefix}-#{$componentName}-#{$cProperty}: #{maintain-quotes($new-value)};
+					} @else {
+						@if type-of($cValue) == map {
+							@each $property, $value in $cValue {
+								$new-value: replace-token-reference($cValue);
+								--#{$prefix}-#{$componentName}-#{$cProperty}-#{$property}: #{maintain-quotes(
+										$new-value
+									)};
+							}
+						}
```

Comment:
@malavikakoppula , I've been trying to figure out why this `@else` condition is here. As far as I can tell it never even gets called. Was there an issue this solved?

Reply:
@vgalatro Forgot to delete that earlier. It's removed now [58ff12b](https://github.com/WPMedia/arc-themes-components/pull/209/commits/58ff12be97fe53f95eda50fcb287635f9f52af39)